frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```
@@ -2,8 +2,10 @@
   "Pipelines": "Pipelines",
   "Select a Builder Image and resource to see if there is a pipeline template available for this runtime.": "Select a Builder Image and resource to see if there is a pipeline template available for this runtime.",
   "The pipeline template for Dockerfiles is not available at this time.": "The pipeline template for Dockerfiles is not available at this time.",
+  "There are no pipeline templates available for {{builderImage}}, current pipeline will be dissociated from application.": "There are no pipeline templates available for {{builderImage}}, current pipeline will be dissociated from application.",
   "There are no pipeline templates available for {{builderImage}} and {{resourceType}} combination.": "There are no pipeline templates available for {{builderImage}} and {{resourceType}} combination.",
   "this Builder Image": "this Builder Image",
+  "Pipeline will be updated to match the builder image.": "Pipeline will be updated to match the builder image.",
```

```suggestion
  "Pipeline will be updated to match the Builder Image.": "Pipeline will be updated to match the Builder Image.",
```
Builder Image is a resource, so a proper noun, must be title case.